src/components/sidebarInternals.svelte

```diff
@@ -72,312 +72,396 @@
 			}
 		}
 	}
+
+	function is_schedule_pdf(chateau_id: string, route_id: string) {
+		if (
+			[
+				'metrolinktrains',
+				'san-diego-mts',
+				'northcountytransitdistrict',
+				'orangecountytransportationauthority',
+				'metro~losangeles'
+			].includes(chateau_id)
+		) {
+			return true;
+		} else {
+			return false;
+		}
+	}
+
+	async function find_schedule_pdf(chateau_id: string, route_id: string) {
```

Comment:
Why is this asyncronous? Why can't this be moved to a component and metro los angeles links be filled in during the component load?

We should show the link to users if they hover the url.